### PR TITLE
Add PIET to the list of Solidity tools

### PR DESCRIPTION
**Pull request by hellraiserinchief**
### Checklist
- [ ] Code compiles correctly
- [x] All tests are passing
- [ ] New tests have been created which fail without the change (if possible)
- [ ] README / documentation was extended, if necessary
- [ ] Changelog entry (if change is visible to the user)
- [ ] Used meaningful commit messages

### CODE REVIEW

**Review by chriseth — DISMISSED** [review]

File: docs/resources.rst [hunk 1]
```diff
@@ -126,6 +126,9 @@ Solidity Tools
 
 * `Universal Mutator <https://github.com/agroce/universalmutator>`_
     A tool for mutation generation, with configurable rules and support for Solidity and Vyper.
+
```
Comment (chriseth) [inline docs/resources.rst]: Our tests are complaining about trailing whitespace here - can you please check?

**Comment by hellraiserinchief** [comment]
All checks seem to pass now

**Review by chriseth — DISMISSED** [review]

**Comment by chriseth** [comment]
Could you please squash it into a single commit?

**Review by erak — DISMISSED** [review]

**Comment by erak** [comment]
@hellraiserinchief Could you please rebase? We just fixed the broken MacOS build on Circle which led to the failing test on this PR.

**Review by erak — APPROVED** [review]

**Comment by erak** [comment]
I'm sorry, but the commits are bit messed up and do appear twice. Did you rebase onto `develop`?

**Review by erak — CHANGES_REQUESTED** [review]
I'm sorry, but the commits are bit messed up and do appear twice. Did you rebase onto develop? Do you need help with this?

**Comment by hellraiserinchief** [comment]
> I'm sorry, but the commits are bit messed up and do appear twice. Did you rebase onto develop? Do you need help with this?

Yeah, help would be welcome, I am new to contributing to OSS

**Comment by hellraiserinchief** [comment]
> > I'm sorry, but the commits are bit messed up and do appear twice. Did you rebase onto develop? Do you need help with this?
> 
> Yeah, help would be welcome, I am new to contributing to OSS

I have tried combined the commits into a single commit, please have a look and advice.

**Review by Marenz — APPROVED** [review]

**Comment by chriseth** [comment]
Sorry, I fear you have to rebase again, the macos version change should be part of a different PR.

**Comment by erak** [comment]
As @chriseth already mentioned, this needs to be rebased again, such that the changes to `.circleci/config.yaml` are not part of the PR. Also, since your working on `develop` of your fork, I'd strongly recommend to create a branch in your forked repository, and then open a PR with our `develop` as the target and your branch as the source. Your current workflow might lead to issues when trying to keep your fork up to date. Please see https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork for further advice.

**Comment by erak** [comment]
@hellraiserinchief Are you able remove the change to `.circleci/config.yml`? Otherwise this PR looks good!

**Review by erak — CHANGES_REQUESTED** [review]
@hellraiserinchief Could you also remove the change to ``.circleci/config.yml``? Otherwise this PR looks good!

**Comment by hellraiserinchief** [comment]
@erak, I will follow your advice and create a new branch to submit a new PR, instead of using the develop branch directly

**Pull request closed without merging**